### PR TITLE
Upgrade for Wagtail 4

### PR DESCRIPTION
**Pull request by nickmoreton**
# Upgrade for Wagtail v4

Solves: https://github.com/torchbox/wagtail-wordpress-import/issues/161

- Testing
    - [x] CI passes
    - [ ] If necessary, tests are added for new or fixed behaviour
    - [x] These changes do not reduce test coverage
- Documentation.
    - [x] This PR adds or updates documentation. But only around code example linting.
    - [ ] Documentation changes are not necessary because:

### CODE REVIEW

File: wagtail_wordpress_import/models.py [hunk 1]
```diff
@@ -27,18 +27,35 @@ def import_wordpress_data(self, data):
...
-            heading="wp data",
-        ),
-        FieldPanel("wp_link", classname="full title"),
-        FieldPanel("wp_block_json", classname="full"),
-        FieldPanel("wp_processed_content", classname="full"),
-        FieldPanel("wp_normalized_styles", classname="full"),
-        FieldPanel("wp_raw_content", classname="full"),
-        FieldPanel("wp_post_meta", classname="full"),
-    ]
+    if WAGTAIL_VERSION >= (3, 0):
```
Comment (nickmoreton) [inline wagtail_wordpress_import/models.py:30]: This is the only change required for Wagtail v4 compatibility.

`classname="full"` has been removed from FieldPanel()

**Review by nickmoreton — COMMENTED** [review]
Just noting changes required for Wagtail v4

File: tox.ini [hunk 2]
```diff
@@ -25,28 +29,24 @@ basepython =
...
 
     django3.1: Django>=3.1,<3.2
     django3.2: Django>=3.2,<3.3
     django4.0: Django>=4.0,<4.1
+    django4.1: Django>=4.1,<4.2
     
     wagtail2.15: wagtail>=2.15,<2.16
     wagtail2.16: wagtail>=2.16,<2.17
     wagtail3.0: wagtail>=3.0,<4.0
+    wagtail4.0: wagtail>=4.0,<4.1
```
Comment (zerolab) [inline tox.ini:41]: suggestion: now that 4.1 is out, may as well add it here. there should be minimal breaking changes (says he hopeful)

Comment (nickmoreton) [reply tox.ini:41]: Testing against Wagtail 4.1 added and passing.

**Pull request merged**